tests/console/postfix_tls.pm

```diff
@@ -42,8 +42,8 @@ sub run {
     systemctl "is-active postfix.service";
 
     # Print service status for debugging
-    script_run "systemctl -l --no-pager status saslauthd.service 2>&1 | tee /dev/$serialdev";
-    script_run "systemctl -l --no-pager status postfix.service 2>&1 | tee /dev/$serialdev";
+    systemctl "-l status saslauthd.service 2>&1 | tee /dev/$serialdev";
+    systemctl "-l status postfix.service 2>&1 | tee /dev/$serialdev";
```

Comment:
This doesn't look good. IMHO we should either leave assert_script_run here, or rewrite code to print same things to serial.

Reply:
what's wrong with it?

Reply:
systemctl wraps calls to systemctl, but here we also execute some other commands, which are related to output, so it's confusing why we call tee in systemctl. Same if we would call multiple commands using systemctl separated by ";". So, I'm not sure if this is an improvement in this case.

Reply:
But I think this is the main point of discussion for this PR. I see the following options:

1) handle `systemctl` as a pure wrapper for `assert_script_run`, i.e. it can be used just as `assert_script_run` including piping the output of systemctl to anything else and potentially including executing a second command after the systemctl one with `;` in between but I would not recommend the latter.
2) just use it in some cases, e.g. whenever `systemctl --no-pager` is used
3) delete the `systemctl` command again

Regarding 1) This is what I was trying to do, I could live with 2) but this will be ambiguous in the future and not easy to understand, I could live with 3) as well.